app/assets/stylesheets/application.css.scss

```diff
@@ -133,7 +133,7 @@ a:hover {
   background:#000;
 }
 
-#username-link a, a:link, a:visited {
+#username-link a, #username-link a:link, #username-link a:visited {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line